usaspending_api/data_load/generic_loaders.py

```diff
@@ -0,0 +1,67 @@
+from usaspending_api.data_load.data_load_helpers import setup_load_lists, setup_mass_load_lists
+
+
+def insert_recipient_locations(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "recipient_location")
+    recipient_location_sql = "INSERT INTO references_location {} VALUES {} RETURNING location_id;".format(
+        columns, values
+    )
+
+    cursor.execute(recipient_location_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_recipients(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "legal_entity")
+    recipient_sql = "INSERT INTO legal_entity {} VALUES {} RETURNING legal_entity_id;".format(columns, values)
+
+    cursor.execute(recipient_sql)
+    return [tup[0] for tup in cursor.fetchall()]
+
+
+def insert_place_of_performance(cursor, load_objects):
+    columns, values = setup_mass_load_lists(load_objects, "place_of_performance_location")
+    recipient_sql = "INSERT INTO references_location {} VALUES {} RETURNING location_id;".format(columns, values)
```

Comment:
Some SQL string end with semicolons, other do not. I think the more common pattern in this project (and broker) is to drop the semicolon. Please standardize all SQL strings without trailing semicolons

Reply:
Sometimes these sql statements are appended before being sent against the database, leaving room for problems when they are appended. Leaving the semicolons in keeps these statements modular.

Reply:
These SQL strings are not being passed around or appended anywhere. They exist and are used in each function.

I've seen more issues in this project caused by trailing semicolons and devs overlooking them when concatenating SQL strings than the inverse.